ui/src/components/SampleGrid/SampleGridTable.css

```diff
@@ -247,7 +247,7 @@ button[disabled] {
   top: 0;
 }
 
-.btn-link {
+.btn-link:not([data-default-styles]) {
```

Comment:
This `color` override, which I assume was for link buttons inside the sample grid, is in fact a global style, so it applies to all `<Button variant="link" />. To avoid breaking anything for now, I'm providing a way to opt-out of this style declaration, like I did with modals.

Reply:
Nice